airflow/www/extensions/init_views.py

```diff
@@ -58,6 +58,7 @@ def init_appbuilder_views(app):
     appbuilder.add_view(views.PoolModelView, "Pools", category="Admin")
     appbuilder.add_view(views.VariableModelView, "Variables", category="Admin")
     appbuilder.add_view(views.XComModelView, "XComs", category="Admin")
+    appbuilder.add_view(views.StatsView, "Stats", category="Admin")
```

Comment:
shouldn't be also a permission that can be granted to access this page for custom roles?

Reply:
@kurtqq good catch, I will add